kroxylicious-operator/src/main/resources/META-INF/fabric8/virtualkafkaclusters.kroxylicious.io-v1.yml

```diff
@@ -76,6 +76,34 @@ spec:
                         maxLength: 253
                         minLength: 1
                         type: string
+                      tls:
+                        description: |
+                          TLS is the TLS configuration for the Ingress. This field is required if
+                          the Ingress definition has Protocol set to "TLS". It is invalid to set this field
+                          if the Protocol field is "TCP".
+                        required: [ "certificateRef" ]
+                        type: object
+                        properties:
+                          certificateRef:
```

Comment:
`clientCertificateRef` ?

Reply:
you mean `serverCertificateRef`?

Reply:
Yes, of course. I was thinking about the equivalent change in the tls object for the KafkaService.